MAUI/ImageEditor/Shape Annotation.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
```

Comment:
change file names.
Save => save
Shape Annotation => shape
Text Annotation => text

Add file names in toc file

Reply:
Changed a file names